### ISSUE TITLE
Shadowmourne

### ISSUE THREAD

**Issue by imnotyou** [TrinityCore/TrinityCore#1619]
The shards should create faster, I think in retail they have a 50% per hit and you could build the full stack in 15 seconds with classes like paladins, in TC2 you have to pray to make a full stack in less than one minute.
##

**Comment by lendan** [comment]
Delete entry in spell_proc_event regarding shadowmourne and let it take proc flags and proc rates from DBC

**Comment by Relina87** [comment]
Info regarding proc chance is available here: http://forums.elitistjerks.com/topic/37305-in-depth-fury-dps-discussion/page-161
For those whom the website may not load I've attached an SS of the section in question.
http://i.imgur.com/TxbzgWk.png

**Comment by Exodius** [comment]
@Relina87 Updating SQL file that was moved to "old" folder is not gonna fix the issue.
You needed to supply fix in a form of an UPDATE so old data can be updated with new, or with a complete DELETE/INSERT statements. In this case, I believe this would be a valid suggested fix, extrapolated from your reference (This is an example of what should be supplied when SQL fix is suggested and then this **may** get pushed and update old data with new).

```sql
UPDATE `spell_proc_event` SET `ppmRate`='12.5', `CustomChance`=75 WHERE `entry`=71903;

**Comment by Keader** [comment]
I called a friend of mine and I recorded him slamming the Shadowmourne in Blizz

With use of Spells: https://www.youtube.com/watch?v=SdqqdjII7i0&feature=youtu.be

No use of Spells: https://www.youtube.com/watch?v=v5LKqa2600g&feature=youtu.be

Recorded on 25/06/2015 on Server Illidan (US)

**Comment by Aokromes** [comment]
Soo. anything here is valid?

**Comment by ariel-** [comment]
Fixed with 2ff855054f52bf2dcf81aa7a7da7bab7f8a99686
